go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/protobuf v1.5.4
 	github.com/google/cadvisor v0.51.0
-	github.com/google/cel-go v0.21.0
+	github.com/google/cel-go v0.21.1-0.20241103013347-3f12ecad39e2
```

Comment:
is it possible to use a tagged cel release?

Reply:
Yes, I requested one last night.  I'll bump this PR to it once it is available.

Reply:
Bumped to v0.22..0, the difference from what was reviewed before is very small: https://github.com/google/cel-go/compare/3f12ecad39e2..v0.22.0